src/backend/distributed/worker/worker_create_or_replace.c

```diff
@@ -75,32 +152,69 @@ WrapCreateOrReplace(const char *sql)
 Datum
 worker_create_or_replace_object(PG_FUNCTION_ARGS)
 {
-	text *sqlStatementText = PG_GETARG_TEXT_P(0);
-	const char *sqlStatement = text_to_cstring(sqlStatementText);
-	Node *parseTree = ParseTreeNode(sqlStatement);
+	List *sqlStatements = NIL;
```

Comment:
can we always pass array instead of maintaining both? I guess that would break rolling upgrades?

Reply:
It is quite an easy refactor to always require an array to be passed. However we mostly don't change the `worker_*` functions signatures much. Guess it would be ok for Citus 11 to require a full cluster upgrade though, but I didn't want to really think much about what could go wrong changing this function, and instead make it work in a backwards compatible manner.

Let me know what your preference is here, I can easily change it to _only_ take arrays.